docs/usage.md

```diff
@@ -0,0 +1,102 @@
+# Usage
+
+This sections describes how to use the CNWAN Reader, i.e. its flags.
+
+It applies to whatever installation method you chose, but if you are running
+the program in a docker container, you should also follow
+[Docker Usage](./docker_usage.md) after reading this.
+
+You should pre-pend `path/to/cnwan-reader` before these commands, or
+`docker run image` if you are running the program inside a docker container.
+
+## Table of Contents
+
+* [CNWAN Adaptor](#cnwan-adaptor)
+* [Metadata Key](#metadata-key)
+* [Service Directory](#service-directory)
+* [Binary Example](#binary-example)
+
+### CNWAN Adaptor
+
+*Adaptors* are external handlers that will receive the events sent by the CNWAN
+Reader and process them.
+
+By default, CNWAN Reader sends data to `localhost:80/cnwan/events`, so it
+expects adaptors to provide a server listening on `localhost:80/cnwan`.  
+In case you have a different endpoint or already have a server listening on
+another host:port or just prefer to use another prefix path - or none at all,
+you can override this behavior with the `-adaptor-api` argument:
+
+For example:
+
+```bash
+--adaptor-api localhost:5588/my/path
+```
+
+Events will be now sent to `localhost:5588/my/path/events`.  
+As an example of no prefix path, `--adaptor-api localhost:8080` will instruct
+the CNWAN Reader to send events on `localhost:8080/events` instead of
+`localhost:8080/cnwan/events`. If a port is not provided, `80` will be used
+as default.
+
+Please follow [OpenAPI Specification](../README.md#openapi-specification)
+to learn more about adaptors and [Example](#example) for a complete usage
+example that includes a CNWAN Adaptor endpoint as well.
+
+### Metadata Key
+
+The CNWAN Reader only reads services that have the provided metadata key.
+
+For example, the following flag
+
+```bash
+--metadata-key prefix/key
```

Comment:
Maybe we should provide an example related to the general CNWAN project, like "cwan.io/traffic-profile", so people coming to the Reader from one of our demos/presos can easily map what this is for.